pom.xml

```diff
@@ -93,7 +93,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>github-api</artifactId>
-            <version>1.90</version>
+            <version>1.95</version>
```

Comment:
Why this dep needs go be updated? Is there any binary compatibility change?

Reply:
It has to do with the security scanner warning about jackson versions. github-api 1.90 has a dependency on [jackson-api 2.7.3](https://github.com/jenkinsci/github-api-plugin/blob/github-api-1.90/pom.xml#L31-L35), whereas 1.95 has its dependency on [2.9.7.1](https://github.com/jenkinsci/github-api-plugin/blob/github-api-1.95/pom.xml#L31-L35).

Reply:
this is not the problem of this plugin, this plugin is only defining minimal required binary compatible version. This is how jenkins dependencies works, hpi is based on maven versioning.

Reply:
FTR I agree with @KostyaSha here. Unless there is a strong need to update (API, integration testing with newer version, and so on), there is no practical need to update